services/ui-src/src/components/export/ExportedModalOverlayReportSection.tsx

```diff
@@ -63,49 +101,113 @@ export function renderStatusIcon(status: boolean) {
-            </Td>
-            <Td>
-              <Text sx={sx.entityList}>
-                {entity.report_planName ?? "Not entered"} <br />
-                {report_programName} <br />
-                {reportingPeriod}
-              </Text>
-            </Td>
-            <Td>
-              <Text>
-                {entity["report_reportingPeriodDiscrepancyExplanation"]
-                  ? entity["report_reportingPeriodDiscrepancyExplanation"]
-                  : "N/A"}
-              </Text>
-            </Td>
-            <Td>
-              <Text>
-                {entity.report_miscellaneousNotes
-                  ? entity.report_miscellaneousNotes
-                  : "N/A"}
-              </Text>
-            </Td>
-          </Tr>
+          <Box>
+            <Tr key={idx}>
+              <Td sx={sx.statusIcon}>
+                <EntityStatusIcon
+                  entity={entity}
+                  isPdf={true}
+                  entityStatus={getInitiativeStatus(report, entity)}
+                />
+              </Td>
+              <Td>
+                <Heading sx={sx.heading} as="h3">
+                  {`${idx + 1}. ${entity.initiative_name}` ?? "Not entered"}{" "}
+                  <br />
+                  <Text sx={sx.headingSubtitle}>
+                    {entity.initiative_wpTopic[0].value}
+                  </Text>
+                </Heading>
+              </Td>
+            </Tr>
+            {/* Depending on what the entity step type is, render its corresponding component */}
+            {entitySteps.map((step, idx) => {
+              switch (step[0].toString()) {
+                case EntityDetailsStepTypes.DEFINE_INITIATIVE:
+                  return (
+                    <Box key={idx}>
+                      <ExportedEntityDetailsOverlaySection
+                        section={section as ModalOverlayReportPageShape}
+                        entity={entity}
+                        entityStep={step}
+                      />
+                    </Box>
+                  );
+                case OverlayModalStepTypes.EVALUATION_PLAN:
+                  return (
+                    <Box key={idx}>
+                      <ExportedOverlayModalReportSection
+                        section={section as OverlayModalPageShape}
+                        entity={entity}
+                        entityStep={step}
+                      />
+                    </Box>
+                  );
+                case OverlayModalStepTypes.FUNDING_SOURCES:
+                  return (
+                    <Box key={idx}>
+                      <ExportedOverlayModalReportSection
+                        section={section as OverlayModalPageShape}
+                        entity={entity}
+                        entityStep={step}
+                      />
+                    </Box>
+                  );
+                // TODO: Once we are tracking the close-out information step, we'll need to add it here
```

Comment:
Do we have a ticket to capture this to make sure we take care of it?

Reply:
@ntsummers1 I created card [CMDCT-3099](https://jiraent.cms.gov/browse/CMDCT-3099) for this